### PR TITLE
Fetch data from API

### PR DESCRIPTION
**Pull request by tajemouti**
## Hi Reviewer 👋 ##
Below is a list of the accomplished tasks while working on this milestone:

1. Created a feature branch for this milestone.
2. Created a new component named Quote.
3. Utilized the useState and useEffect hooks.
4. Fetched data from the Ninjas Quotes API using the fetch function.
5. Implemented a loading state while fetching data.
6. Handled errors that might occur during the API request.
7. Randomly selected a quote from the fetched data and displayed it along with the author.
8. Integrated the Quote component into the application alongside the existing Calculator component.
9. Added some basic styling to the application.

### CODE REVIEW

File: src/components/Quote.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+
+const Quote = () => {
+  const [quote, setQuote] = useState({});
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
```
Comment (OyePriscilla) [inline src/components/Quote.js:8]: - [x] The implementation of **Quote.js** is ok in accordance to the requirement 👍, but <Quotes> component should be imported into the main component as specifically mentioned in the requirement, kindly follow the procedure given. 👇 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @tajemouti ✔️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Created a new component for displaying a quote ✔️ 
Fetched data inside of a new component ✔️ 
Show a loading and error states ✔️ 
Imported the new component into Calclator.js component ❎ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Quote.js (hunk 1 above)
Comment (tajemouti) [reply src/components/Quote.js:8]: Hi @OyePriscilla,
I will do that. Thank you for the fast and insightful review.
Have a great time ahead.

Comment (tajemouti) [reply src/components/Quote.js:8]: Hi again @OyePriscilla,

Upon revisiting the milestone requirements and carefully considering your feedback, there might be a slight misinterpretation of the requirement regarding the placement of the Quote component. The requirement states: "Import the new component into the main component (next to the Calculator). It should be on the same page as the Calculator."
In light of this, I made sure to adhere to the requirement by importing the Quote component into the main component, which is the 'App' component. This placement effectively fulfills the intent of having the Quote component on the same page as the Calculator.
To clarify, the main component that was referred to is 'App' and not the 'Calculator' component. By including the Quote component in 'App', I've met the requirement's intent of having both the Calculator and Quote components on the same page.
I apologize for the confusion, and I hope this explanation clarifies the implementation approach I've taken. Your feedback has been valuable, and I'm going to request a re-review with the requested change anyway, but I'm confident that my code is aligned with the milestone's requirements.
Thank you once again for your time and attention. I look forward to any further guidance or suggestions you might have.

Cheers

**Review by ademibowale — APPROVED** [review]
Hi @tajemouti  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️
- There is a loading state✔️
- A <Quotes> component has been created
- The useEffect hook has been used correctly ✔️
- There is an error state ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**